### PR TITLE
Functions to store and compute climatological weights

### PR DESCRIPTION
**Pull request by ladc**
This implements the functions save_weights and calc_clim_weights which save the current and compute the climatological weights (a running mean of the weights of the past n days, where typically n=30. These climatological weights are used in the scale- and skill-dependent blending of STEPS with NWP forecasts, see #212,  which will be part of pysteps v2.0, see #216.
The current weights, based on the correlations per cascade level, are regressed towards these climatological weights in the course of the forecast.

### CODE REVIEW

File: pysteps/blending/clim.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    Add the current NWP weights to update today's daily average weight. If the
+    day is over, update the list of daily average weights covering a rolling
+    window.
+
+    Parameters
+    ----------
+    current_weights: array-like
+      Array of shape [model, scale_level, ...]
+      containing the current weights of the different NWP models per cascade level.
+    outdir_path: string
```
Comment (RubenImhoff) [inline pysteps/blending/clim.py]: validtime still misses here. What kind of date string should it be?

**Review by RubenImhoff — COMMENTED** [review]
Nice work, @ladc!! 
I have one question left and I made some changes to make all the tests working. :)

File: pysteps/blending/clim.py (hunk 1 above)
Comment (ladc) [reply pysteps/blending/clim.py]: Thanks for the feedback! I looked into using numpy's built-in datetime64 type but got discouraged by the apparent lack of documentation. So I'll stick to datetime instead.

**Review by ladc — COMMENTED** [review]
I guess this works now!

**Pull request merged**